vite.config.ts

```diff
@@ -11,7 +11,12 @@ import svgr from "vite-plugin-svgr";
 import { configDefaults } from "vitest/config";
 // https://vitejs.dev/config/
 export default defineConfig(() => ({
-  plugins: [svgr(), react(), cesium({ rebuildCesium: true }), dts({ rollupTypes: true })],
+  plugins: [
+    svgr(),
+    react(),
+    cesium({ rebuildCesium: true }),
+    dts({ rollupTypes: true, exclude: ["./src/env.d.ts"] }),
```

Comment:
```suggestion
    dts({ rollupTypes: true }),
```
I thought this is necessary, but why did you exclude it?